src/main/java/org/candlepin/model/ContentCurator.java

```diff
@@ -158,7 +158,7 @@ public Map<String, Set<String>> getProductsReferencingContent(Collection<String>
     }
 
     /**
-     * Fetches a set consisting of the content of the  products specified by the given UUIDs. If the
+     * Fetches a set consisting of the content of the products specified by the given UUIDs. If the
```

Comment:
Update "Fetches a set" to "Fetches a collection" or maybe "Fetches a list".

It looks like this could also apply to the following methods:
ProductCurator.getChildrenProductsOfProductsByUuids

Reply:
fixed